recipes-extended/aamp/aamp_git.bb

```diff
@@ -44,6 +44,11 @@ EXTRA_OECMAKE += "  -DCMAKE_WPEFRAMEWORK_REQUIRED=1 "
 EXTRA_OECMAKE += "${@bb.utils.contains('DISTRO_FEATURES', 'sec_manager', ' -DCMAKE_USE_SECMANAGER=1 ', '', d)}"
 EXTRA_OECMAKE += " -DCMAKE_WPEWEBKIT_WATERMARK_JSBINDINGS=1 "
 
+EXTRA_OECMAKE:append += "${@bb.utils.contains('DISTRO_FEATURES', 'flex2_rdk' , ' -DCMAKE_SUBTEC_CC_ENABLED=1 -DCMAKE_FLEX2_RDK=1', '', d)}"
```

Comment:
we should remove the flex2_rdk flag. This is a flag from RDK-V where flex2_rdk meant ENT OS stack. In RDK-E, everything is ENT OS. Pls check and enable it.